infra/src/main/scala/ch/epfl/scala/utils/DoobieUtils.scala

```diff
@@ -200,7 +218,7 @@ object DoobieUtils {
         }
 
     implicit val projectReader: Read[NewProject] =
-      Read[(Organization, Repository, Option[Instant], GithubStatus, Option[GithubInfo], NewProject.DataForm)]
+      Read[(Organization, Repository, Option[Instant], GithubStatus, Option[GithubInfo], Option[NewProject.DataForm])]
```

Comment:
I guess it's preparatory to change dataform to Option[DataForm] on Project

Reply:
It's because insertRelease inserts the project ref and status but not the dataForm, so dataForm can be null in the DB.

But yes I hope we can make an option.